limits/util.py

```diff
@@ -6,9 +6,12 @@
 
 from .limits import GRANULARITIES
 
-
+SINGLE_EXPR = re.compile(
```

Comment:
This could just be a string which is used to construct the `EXPR` compiled regular expression.